postgres/assets/configuration/spec.yaml

```diff
@@ -356,6 +356,7 @@ files:
               - "model"
               - "msdb"
               - "cloudsqladmin"
+              - "rdsadmin"
```

Comment:
Is this the right place to edit this? `example` key doesn't seem like it will actually change the default behavior. Will it?

---

Separately: I don't want to block this PR specifically, but seeing how this is done, this pattern is suboptimal for user experience. 

Example: Let's say a customer has a database `audit` they want to ignore. Their workflow might look like this:

1. See that there is this `database_autodiscovery.exclude` option. There is also an `ignore_databases` option. They have to add that database to both.
2. If they just add their own database here, i.e. `["audit"]`, they override the defaults and `rdsadmin` will be autodiscovered again.
3. Let's say they get it correct and they modify their config to add all of the defaults plus their `audit` database. 3 months down the road, we add another database to the default. Their configs will overwrite that default. 

So I think the best option is to separate the default ignored databases and the user-specified ignored databases. Wdyt?

Reply:
I do agree with what you described. But we will need to add a bit more thoughts to this, mainly because the default databases in `ignore_databases` and `database_autodiscovery.exclude` can be categorized into 2 different categories.
1. Databases that we don't have access to - `rdsadmin, cloudsqladmin, azure_maintenance` - these database should be auto excluded for both monitoring and auto discovery.
2. Databases usually not worth monitoring but could be monitored - `template%, postgres, msdb` - they are recommended to be ignored but user should be able to override the default.